### PR TITLE
Adding capability to add a custom logo

### PR DESCRIPTION
**Pull request by RAVEENSR**
## Purpose
Add ability to add a custom logo for the analytics-dashboard

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java [hunk 1]
```diff
@@ -45,6 +45,12 @@
     @Element(description = "Report generation configurations")
     Map<String, Object> reportGeneration = Collections.emptyMap();
 
+    @Element(description = "Map of roles list")
```
Comment (TanyaM) [inline components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java]: Is this description correct?

File: components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java [hunk 2]
```diff
@@ -45,6 +45,12 @@
...
+    public String themeConfigProviderClass = "org.wso2.carbon.dashboards.core.DefaultDashboardThemeConfigProvider";
+
+    @Element(description = "Map of roles list")
```
Comment (TanyaM) [inline components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java]: Description?

File: components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java (hunk 1 above)
Comment (RAVEENSR) [reply components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java]: Fixed in 8370d4d40671746b5d46d947c1daad2f0fffd5a5 .

File: components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java (hunk 2 above)
Comment (RAVEENSR) [reply components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java]: Fixed in 8370d4d40671746b5d46d947c1daad2f0fffd5a5 .

**Pull request merged**